From 84c8769b52c63cce8a8a413e5fcbad1a65681d46 Mon Sep 17 00:00:00 2001
From: Joshua Watt <JPEW.hacker@gmail.com>
Date: Tue, 2 Jul 2019 13:06:30 -0500
Subject: [PATCH] Fix include path separator

Changes the include path separator for Windows builds to use "/" instead
of "\". Windows has no problems with using a forward slash as a path
separator, but using a backslash causes problems when attempting to
cross compile for other platforms (for example, when trying to cross
compile for MinGW from Linux).

Signed-off-by: Joshua Watt <JPEWhacker@gmail.com>
Upstream-Status: Submitted [https://sourceware.org/ml/bzip2-devel/2019-q3/msg00004.html]
---
 bzip2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bzip2.c b/bzip2.c
index e362c65..be3b3be 100644
--- a/bzip2.c
+++ b/bzip2.c
@@ -128,7 +128,7 @@
 #if BZ_LCCWIN32
 #   include <io.h>
 #   include <fcntl.h>
-#   include <sys\stat.h>
+#   include <sys/stat.h>
 
 #   define NORETURN       /**/
 #   define PATH_SEP       '\\'
-- 
2.21.0

